frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTable.tsx

```diff
@@ -210,17 +254,30 @@ const PipelineRunTable: React.FC<PipelineRunTableProps> = ({
             checkboxProps={{
               isChecked: isSelected(run.run_id),
               onToggle: () => toggleSelection(run.run_id),
+              tdProps: { isStickyColumn: !!isExperimentsEnabled, stickyMinWidth: '45px' },
             }}
             onDelete={() => {
               setSelectedIds([run.run_id]);
               setIsDeleteModalOpen(true);
             }}
             run={run}
-            hasExperiments={!(isExperimentsAvailable && experimentId)}
+            hasExperiments={!isExperimentsEnabled}
+            customCells={metricsColumnNames.map((metricName: string) => (
+              <Td key={metricName} dataLabel={metricName}>
+                {!artifactsLoaded && !artifactsError ? (
+                  <Skeleton />
```

Comment:
this skeleton seems to go away before a value is found. i am seeing
1. refresh
2. cell skeleton
3. `-`
4. 123

Reply:
This should be fixed now.